src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -30,9 +31,12 @@
 
 if is_torch_available():
     import torch
+    import torch.nn as nn
 
 if is_torchao_available():
+    from torchao.dtypes import AffineQuantizedTensor
     from torchao.quantization import quantize_
+    from torchao.quantization.linear_activation_quantized_tensor import LinearActivationQuantizedTensor
```

Comment:
small nit that I didn't check last time but can we move these imports inside the function where we are using them ? This way, transformers import will be faster